biosimulators_utils/config.py

```diff
@@ -117,7 +129,7 @@ def __init__(self,
             COLLECT_SED_DOCUMENT_RESULTS (:obj:`bool`, optional): whether to assemble an in memory data structure with all of the
                 simulation results of SED documents
             SAVE_PLOT_DATA (:obj:`bool`, optional): whether to save data for plots alongside data for reports in CSV/HDF5 files
-            REPORT_FORMATS (:obj:`list` of :obj:`str`, optional): default formats to generate reports in
+            REPORT_FORMATS (:obj:`list` of :obj:`str`, optional): default formats to generate reports in-->defaults to 'csv'
```

Comment:
This PR sets the default to `None`

Reply:
I originally was generating the value of `REPORT_FORMATS` to have a default constructor setting of `[ReportFormat.csv]` (formats are of `type(enum.Enum)`), or even `[ReportFormat.h5]`. Our good friend `SonarCloud` complained about this definition, which makes sense as the definition is also performing a very basic operation other than just instantiation. This doc string annotation was from when I had it in the definition and must have missed it.

Reply:
After review, that doc string is actually as I intended it to be. Following the logic of the constructor, if there is no value for the `REPORT_FORMATS` attribute upon instantiation, it defaults to `'csv'`. As per the constructor: `self.REPORT_FORMATS = REPORT_FORMATS or [ReportFormat.csv]`, which evaluates roughly as:

`if not REPORT_FORMATS:`
`    REPORT_FORMATS = [ReportFormat.csv]`